README.md

```diff
@@ -210,6 +210,19 @@ A `socket` or `host` parameter can be specified to connect through a unix socket
 DATABASE_URL="postgres://username:password@/database_name?socket=/var/run/postgresql"
 ```
 
+For passwordless authentication like Postgres [peer auth-method](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html) the URL can be shortened:
+
+```sh
+DATABASE_URL="postgres://username/database_name?socket=/var/run/postgresql"
```

Comment:
Does this work? I would have expected needing at least `postgres://username@/database_name?socket=/var/run/postgresql` (with `@`, to differentiate it from a hostname)

Reply:
Yes, i tested it with Postgres 14.9.

Reply:
@amacneil This only works with a local connection with the [`peer` authentication method](https://www.postgresql.org/docs/current/auth-peer.html), because in that specific mode, Postgres isn't using the hostname or username from the connection string, as it's connecting locally (so hostname is irrelevant) and `peer` auth relies on Postgres getting the OS username, so the username from the connection string is irrelevant.

In other words, this works:

```
postgres@e8f183dbe0dd:/tmp$ ./dbmate -u postgres:///dbmate_test status
[ ] 20231115033218_test.sql

Applied: 0
Pending: 1
```

Notice that I'm doing this as user `postgres`, which happens to be a valid database user.  If I try this as user `root` that isn't a valid user in this Postgres database, we get this error:

```
root@e8f183dbe0dd:/tmp# ./dbmate -u postgres:///dbmate_test status
Error: pq: role "root" does not exist
```